ush/python/pygfs/jedi/jedi.py

```diff
@@ -216,17 +214,27 @@ def get_bias_dict(self, task_config: AttrDict) -> Dict[str, Any]:
                 obfile = ob['obs bias']['input file']
                 obdir = os.path.dirname(obfile)
                 basename = os.path.basename(obfile)
-                prefix = '.'.join(basename.split('.')[:-2])
-                for file in ['satbias.nc', 'satbias_cov.nc', 'tlapse.txt']:
-                    bfile = f"{prefix}.{file}"
-                    copylist.append([os.path.join(task_config.COM_ATMOS_ANALYSIS_PREV, bfile), os.path.join(obdir, bfile)])
-                    # TODO: Why is this specific to ATMOS?
+                prefix = '.'.join(basename.split('.')[:-3])
+                bfile = f"{prefix}.rad_varbc_params.tar"
+                radtar = os.path.join(obdir, bfile)
+                copylist.append([os.path.join(task_config.COM_ATMOS_ANALYSIS_PREV, bfile), radtar])
```

Comment:
I'm not saying we have to do it in this PR, but this is named `get_bias` and is in the Jedi class, but it is hard coded for atmospheric radiance bias correction. We need to think of a way to make it more generic for aircraft, but also other components (aerosol, possibly marine, etc.)

Could we perhaps pass in 1) an additional argument to specify `bfile` that way we can call this twice for radiance and aircraft VarBC for the atmosphere and 2) ensure that instead of `task.config.COM_ATMOS_ANALYSIS_PREV` we define it more generally as `task_config.VarBcDir` or something like that, and let the calling script/function/method set `task_config.VarBcDir equal to the appropriate `COM_*_ANALYSIS_PREV`?

Reply:
@CoryMartin-NOAA , let's see what can be done to achieve the generalization you have in mind.  As you say, this will be iterative in that this PR may not get us to the ultimate end we want but we can at least try to take a few steps toward the goal.  With whom should I collaborate on the EMC JEDI team?

Reply:
Initial attempt to generalize processing of variational bias correction files committed at 666d7d6d.  Feedback welcomed!

Reply:
what you've added looks good to me @RussTreadon-NOAA

Reply:
Found it necessary to move `VarBcDir` inside `initialize_analysis` because COM_*_ANALYSIS_PREV is not defined for every atmospheric variational and local ensemble DA job.  Done at 1e4a2843.

Reply:
Run g-w CI using `feature/radbcor` at [`386313b`](https://github.com/RussTreadon-NOAA/global-workflow/commit/386313bad05fd6f35ea7d10f3f9dd672ff94243e) on Hera.  The following g-w CI configurations are being run
- C96C48_hybatmDA
- C96C48_ufs_hybatmDA
- C96C48_hybatmaerosnowDA
- C48mx500_3DVarAOWCDA

While cycling continues, each of the above tests have completed DA jobs for the first full cycle. 

All `test_gdasapp` tests _Passed_ using g-w `feature/radbcor` at [`386313b`](https://github.com/RussTreadon-NOAA/global-workflow/commit/386313bad05fd6f35ea7d10f3f9dd672ff94243e) and GDASApp `feature/radbcor` at [`554c55a`](https://github.com/RussTreadon-NOAA/GDASApp/commit/554c55a6ad1a4fc7d5868122a9b0147af3b300a2)

Reply:
Resolving conversation as I think the reviewers' useful comments have been incorporated into this PR.